src/Handlers/SuppressHandler.php

```diff
@@ -2,39 +2,108 @@
 
 namespace Psalm\LaravelPlugin\Handlers;
 
-use Illuminate\Console\Command;
-use Illuminate\Foundation\Http\FormRequest;
-use Illuminate\Notifications\Notification;
 use Psalm\Plugin\EventHandler\AfterClassLikeVisitInterface;
 use Psalm\Plugin\EventHandler\Event\AfterClassLikeVisitEvent;
+use Psalm\Storage\ClassLikeStorage;
+use Psalm\Storage\MethodStorage;
+use Psalm\Storage\PropertyStorage;
+use function array_intersect;
 use function in_array;
 
 class SuppressHandler implements AfterClassLikeVisitInterface
 {
+    /**
+     * @var array<string, list<class-string>>
+     */
+    private const BY_CLASS = [
+        'UnusedClass' => [
```

Comment:
I wonder if these are still emitted if the `bootstrap/app.php` file is analyzed? Thinking out loud..

Reply:
Good point, `bootstrap/app.php` *does* use `App\Console\Kernel`, `App\Exceptions\Handler` and `App\Http\Kernel`. And some of the other classes *are* used in `config/app.php`.

So maybe it is worth adding those directories to the `tests/(laravel|lumen)-test-psalm.xml` configs. 🤔 

In my project, I have configured `<projectFiles>` likes this, probably due to too many errors (and some of this could explain some of the weird errors I have experienced):

```xml
    <projectFiles>
        <directory name="."/>
        <ignoreFiles allowMissingFiles="true">
            <directory name="app/Http/Middleware"/>
            <directory name="bootstrap/cache"/>
            <directory name="cache"/>
            <directory name="config"/>
            <directory name="database"/>
            <directory name="node_modules"/>
            <directory name="routes"/>
            <directory name="storage"/>
            <directory name="tests"/>
            <directory name="vendor"/>
            <directory name="vendor-bin"/>
            <file name="bootstrap/app.php"/>
            <file name="public/index.php"/>
            <file name="server.php"/>
        </ignoreFiles>
    </projectFiles>
```

Reply:
yep, didn't view that as a blocker for this PR, but something we should consider!